### PR TITLE
Add 1 endpoint in Cosmic Counts API

### PR DESCRIPTION
**Pull request by ersinciftci**
POST /cosmic-counts/fetch

### CODE REVIEW

**Comment by ersinciftci** [comment]
[Implement COSMIC API [API Spec Ready]](https://trello.com/c/Cii9Bmoz/6-implement-cosmic-api-api-spec-ready)

File: web/src/main/java/org/cbioportal/web/MrnaPercentileController.java [hunk 1]
```diff
@@ -34,9 +34,9 @@
     private MrnaPercentileService mrnaPercentileService;
 
     @RequestMapping(value = "/genetic-profiles/{geneticProfileId}/mrna-percentile/fetch", method = RequestMethod.POST,
-        produces = MediaType.APPLICATION_JSON_VALUE)
+        consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
     @ApiOperation("Get mRNA expression percentiles for list of genes for a sample")
-    public ResponseEntity<List<MrnaPercentile>> getMrnaPercentile(
+    public ResponseEntity<List<MrnaPercentile>> fetchMrnaPercentile(
```
Comment (onursumer) [inline web/src/main/java/org/cbioportal/web/MrnaPercentileController.java]: @ersinciftci is this related to Cosmic Count?

Comment (ersinciftci) [inline web/src/main/java/org/cbioportal/web/MrnaPercentileController.java]: No, just a small method name change in Mrna Percentile API that wouldn't worth creating a separate PR.

File: persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/CosmicCountMyBatisRepositoryTest.java [hunk 2]
```diff
@@ -21,50 +21,23 @@
...
-	public void getCOSMICCountsByKeywordsReusedKeyword() {
-		List<String> keywords = new LinkedList<>();
-		keywords.add("NOC2L truncating");
-		List<CosmicCount> result = cosmicCountMyBatisRepository.getCOSMICCountsByKeywords(keywords);
+		
+		List<CosmicMutation> result = cosmicCountMyBatisRepository.fetchCosmicCountsByKeywords(keywords);
 		Assert.assertEquals(2, result.size());
+		CosmicMutation cosmicMutation1 = result.get(0);
+		Assert.assertEquals("3677745", cosmicMutation1.getCosmicMutationId());
+        Assert.assertEquals("D45A", cosmicMutation1.getProteinChange());
```
Comment (onursumer) [inline persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/CosmicCountMyBatisRepositoryTest.java]: a minor indentation mismatch here
